randovania/games/samus_returns/json_data/Area 2 - Exterior.txt

```diff
@@ -214,7 +214,15 @@ Extra - asset_id: collision_camera_006
   > Tunnel to Spring Ball Tutorial
       Lay Any Bomb
   > Event - Arachnus
-      Trivial
+      All of the following:
+          Any of the following:
+              # Energy
+              Combat (Intermediate) or Normal Damage ≥ 199
+              Combat (Beginner) and Normal Damage ≥ 99
+          Any of the following:
+              # Damage
+              Missile ≥ 30 and Lay Bomb
+              Combat (Beginner) and Lay Power Bomb
```

Comment:
even tho i played msr recentlyish, i cant remember arachnus at all really so
- what's your reasoning for locking PB behind combat beginner?
- what are the circumstances leading to 30missiles+bomb?

Reply:
- Partly because pbs end the fight instantly
- 30 missiles is roughly what you need to finish the fight without needing to use the beam. Dunno if that's even worth accounting for